src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfMapperService.java

```diff
@@ -0,0 +1,340 @@
+package uk.gov.hmcts.reform.et.syaapi.service.pdf;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.et.common.model.ccd.Address;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.items.RespondentSumTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.ClaimantOtherType;
+import uk.gov.hmcts.et.common.model.ccd.types.NewEmploymentType;
+import uk.gov.hmcts.et.common.model.ccd.types.RepresentedTypeC;
+import uk.gov.hmcts.et.common.model.ccd.types.RespondentSumType;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.NO;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.YES;
+
+/**
+ * Maps Case Data attributes to fields within the PDF template.
+ * Inputs that are accepted from the ET1 form can then be mapped to
+ * the corresponding questions within the template PDF Provided
+ */
+@Service
+public class PdfMapperService {
+    private static final  Map<String, String> TITLES = Map.of(
+        "Mr", PdfMapperConstants.Q1_TITLE_MR,
+        "Mrs", PdfMapperConstants.Q1_TITLE_MRS,
+        "Miss", PdfMapperConstants.Q1_TITLE_MISS,
+        "Ms", PdfMapperConstants.Q1_TITLE_MS
+    );
+    private static final Map<String, String> TITLE_MAP = Map.of(
+        "Mr", "Mister",
+        "Mrs", "Mrs",
+        "Miss", "Mis",
+        "Ms", "Ms"
+    );
+    private static final String[] ADDRESS_PREFIX = {
+        "2.2",
+        "2.5 R2",
+        "2.7 R3",
+        "13 R4",
+        "13 R5"
+    };
+    private static final String REP_ADDRESS_PREFIX = "11.3 Representative's address:";
+    private static final String CLAIMANT_ADDRESS_PREFIX = "1.5";
+    private static final int MULTIPLE_RESPONDENTS = 2;
+    private static final int MAX_RESPONDENTS = 5;
+    private static final String[] ACAS_PREFIX = {
+        "2.3",
+        "2.6",
+        "2.8",
+        "13 R4",
+        "13 R5"
+    };
+
+    /**
+     * Maps the parameters within case data to the inputs of the PDF Template.
+     *
+     * @param caseData          the case data that is to be mapped to the inputs in the PDF Template.
+     * @return                  a Map containing the values from case data with the
+     *                          corresponding PDF input labels as a key.
+     */
+    public Map<String, Optional<String>> mapHeadersToPdf(CaseData caseData) {
+        ConcurrentHashMap<String, Optional<String>> printFields = new ConcurrentHashMap<>();
+        printFields.put(PdfMapperConstants.TRIBUNAL_OFFICE, Optional.ofNullable(caseData.getManagingOffice()));
+        printFields.put(PdfMapperConstants.CASE_NUMBER, Optional.ofNullable(caseData.getCcdID()));
+        printFields.put(PdfMapperConstants.DATE_RECEIVED, Optional.ofNullable(caseData.getReceiptDate()));
```

Comment:
Given we're using Optional.ofNullable so much, let's add a static import to drop "Optional." from the code :)